src/bindgen.rs

```diff
@@ -136,6 +136,7 @@ pub fn wasm_bindgen_build(
         };
         let target_arg = match target {
             "nodejs" => "--nodejs",
+            "no-modules" => "--no-modules",
```

Comment:
Not in this PR, but as a follow up issue, this seems like a good candidate for an `enum` with `FromStr` and `Display` impls like how we do with `BuildMode` and `Access`